apps/assisted-disconnected-ui/src/components/ResetSingleClusterModal.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {
+  Button,
+  Modal,
+  ModalVariant,
+  ButtonVariant,
+  Text,
+  TextContent,
+} from '@patternfly/react-core';
+import { useModalDialogsContext, ClustersService } from '@openshift-assisted/ui-lib/ocm';
+import { getApiErrorMessage, handleApiError, ErrorState } from '@openshift-assisted/ui-lib/common';
+import { useNavigate } from 'react-router-dom-v5-compat';
+
+const ResetSingleClusterModal: React.FC = () => {
+  const navigate = useNavigate();
+  const [error, setError] = React.useState<{ title: string; message: string } | null>(null);
```

Comment:
no need to spell the `null`
```suggestion
  const [error, setError] = React.useState<{ title: string; message: string }>();
```